### ISSUE TITLE
🐞 Bug: Wrong display when input field is empty when login

### ISSUE THREAD

**Issue by upsaurav12** [CircuitVerse/CircuitVerse#4397]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

in login when we Click on login button is showing wrong message unlike when we do Sign up

### Steps To Reproduce

1.Go to https://circuitverse.org/users/sign_in
2. Switch to login if not 
3. press in submit with empty field

### Expected Behavior

It should display like when we do same with sign up
![image](https://github.com/CircuitVerse/CircuitVerse/assets/116784047/1614dcd4-3a55-427b-9398-ba474207664f)

### Screenshots

It is display the same when the reCAPTCHA not verified properly or incorrect 
![image](https://github.com/CircuitVerse/CircuitVerse/assets/116784047/a79bdab7-1b60-4ad0-a745-0d212b87506b)

### Device Information [optional]

```markdown
- OS: windows
- Browser: chrome 
- version:120
```

### Are you working on this issue?

Yes

**Comment by upsaurav12** [comment]
pls assing this to me

**Comment by tanmoysrt** [comment]
1. In the maximum scenario, the re-captcha is verified first and then the user inputs.
If you confirm the captcha and then submit the blank form it will show.
![image](https://github.com/CircuitVerse/CircuitVerse/assets/57363826/ca1e1502-0857-444f-8829-0d9652656fe9)
2. However, We need to make the input fileds `required` to prevent user from submitting blank forms.
@upsaurav12 what do you think about that?

**Comment by ShubhaamTiwary** [comment]
Yes @tanmoysrt , I think we need to show input field validation as the error message and need to make fields required.

**Comment by upsaurav12** [comment]
@tanmoysrt i tried required in input field as shown 
1. ![image](https://github.com/CircuitVerse/CircuitVerse/assets/116784047/4c83e7f4-9ae8-4f0a-9b3d-80da8c975fa7)
would it be fine to show message or some input error same as in sign in ??

2. should we disable the login button once the input field , password , and recaptcha verified then only the login button will enable??

**Comment by tanmoysrt** [comment]
> @tanmoysrt i tried required in input field as shown
> 
> 1. ![image](https://private-user-images.githubusercontent.com/116784047/292636059-4c83e7f4-9ae8-4f0a-9b3d-80da8c975fa7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMzNTE4MjcsIm5iZiI6MTcwMzM1MTUyNywicGF0aCI6Ii8xMTY3ODQwNDcvMjkyNjM2MDU5LTRjODNlN2Y0LTlhZTgtNGYwYS05YjNkLTgwZGE4Yzk3NWZhNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIyM1QxNzEyMDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZGJhYzRiMmEzOTljNTQxMDhhNzYzZjMwZjU0ZjNjNjE0M2M4MTA5ODEwY2U2M2RjYTgxOTM3MzE3ODQ5NjQ0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.te1y_yGoc2nOt8xeVIfnKflqHuRu-pfThex4fyDb7uA)
>    would it be fine to show message or some input error same as in sign in ??
> 2. should we disable the login button once the input field , password , and recaptcha verified then only the login button will enable??

you can go for 2nd choice, you don't need to check whether recaptcha has been verified
just check if the input fields is not empty, that's enough

**Comment by upsaurav12** [comment]
@tanmoysrt i implemented the idea that u suggest 

https://github.com/CircuitVerse/CircuitVerse/assets/116784047/105850e0-91d6-4e3f-80f8-e0159ab0a6ed



if there is any other changes to be made

**Comment by upsaurav12** [comment]
@tachyons @tanmoysrt pls review the changes that i made

**Comment by Kiran-pande-30** [comment]
Can we also change sign response messages?
![image](https://github.com/CircuitVerse/CircuitVerse/assets/105718863/ae227f26-ccc0-491d-93a2-17ec263c8b11)
